tdrs-backend/tdpservice/parsers/schema_defs/tanf/t4.py

```diff
@@ -0,0 +1,48 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import SchemaManager
+from ...fields import Field
+from ...row_schema import RowSchema
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T4
+
+
+t4 = SchemaManager(
+      schemas=[
+        RowSchema(
+          model=TANF_T4,
+          preparsing_validators=[
+              validators.hasLength(71),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+                    required=True, validators=[]),
+              Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
+                    required=True, validators=[]),
+              Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                    required=True, validators=[]),
+              Field(item="2", name='COUNTY_FIPS_CODE', type='number', startIndex=19, endIndex=22,
```

Comment:
string, because county fips code can have leading zeros. we may need to check this is the case for the other sections as well. should be string.